### DISCUSSION TITLE
New Widget Proposal - Star rating

### DISCUSSION THREAD

**Discussion by gondar00** [ToolJet/ToolJet#524] [Ideas]
# 🌱 New Widget - Star rating

[//]: # (Link to the pitch made for the cycle)
[//]: # (Github Repos indicate the repos affected or the requirement to create new repos for this project)
[//]: # (Add a link to the asana board when the project starts)
[//]: # (Every proposal requires some sort of a repository/gist where a POC has been done)

|  |  |
|-|-|
| Author           | Gandharv  (github.com/gondar00)     |
| Pitch   | github.com/ToolJet/ToolJet/issues/{todo-update-issue-number-here}     |
| Github Repos   |      |
| Trello           |      |

## 📃 Problem Statement

[//]: # (💥 High level detail of the problem to be solved, assumptions made, etc that pertains to the project)

### 💼 Migration Plan

[//]: # (In case we're migrating data)

## ❗ Risks
[//]: # (Unknowns, places where things could potentially go wrong ex: availability of third party vendor, security, etc)

## ✅ "I've read this" section
[//]: # (Fill the table below the people you want to review this proposal. Reviewers indicate whether you have gone through the proposal and approve it if you're satisfied with the level of detail.)

| Name | Status |
|-|-|
|  |  Not Read    |
|  |  Not Read    |

**Comment by shivangT** [comment]
Hey @gondar00 

Even I've listed this separately in the feature sheet I maintain. 
Got some UI examples to share before we build this. Let me know if you need some suggestions for the same.
Thanks

**Reply by gondar00** [reply]
Hey @shivangT this feature is already in place in existing widgets resolved in #535 

Would be nice to look at the examples you have for this and we can re-visit this widget :)
